Fix P-GPS to work with other partitions in external flash (#9860)
nrfconnect/sdk-nrf: samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

``nrf9160dk_nrf9160_ns@1_0_1``

This sample also supports placement of the MCUboot secondary partition in external flash for the nRF9160 development kit version 0.14.0 and higher.
To enable this, add the following parameter to your build command:

> ``-DOVERLAY_CONFIG=overlay_mcuboot_ext_flash.conf``

Reviewer: when looking for the pgps external flash overlay, i noticed that it is documented in the [PGPS readme](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud_pgps.html), but the overlay file lives only in the MMSS sample.  the PGPS readme doesn't tell the user where to find the file.
is this intentional/desired?
also it does not mention that the overlay file found in the MMSS project is needed:
```
chosen {
		nordic,pm-ext-flash = &mx25r64;
	};
	
```

Author: I'll fix that.  I put it there by mistake back in November in a previous PR, and no one caught it then.

> and specify your development kit version as indicated above.

Reviewer 2: ```suggestion
Then specify your development kit version as described earlier.
```
